app/assets/stylesheets/application.scss

```diff
@@ -14,4 +14,5 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "lists";
+@import "gift_requests";
```

Comment:
I needed to import this line for the form styles to propagate to the wishlist forms as well. If I remove the import, the changes will only be applied to the gift request forms since they're being applied on a controller level basis. Let me know if that makes sense or if I'm missing something?